quickstart/101-firewall-standard/main.tf

```diff
@@ -0,0 +1,106 @@
+terraform {
```

Comment:
What I meant by my previous comment was that this block should include this property to ensure that the right version of Terraform is being used and can correctly interpret the var.blah syntax:

required_version = ">=0.12"

If you have tested using 1.0, you should require that version instead of 0.12.

These changes are good as well.